sorting/tim_sort/tim_sort.cpp

```diff
@@ -21,27 +21,61 @@
 #include <bits/stdc++.h> 
```

Comment:
Replace this with specific libs that are needed instead of all of em.
Greatly improves compile time!

Reply:
Okay